### PR TITLE
LTD-2898-remove-non-firearms-ff 

### PR DESCRIPTION
**Pull request by depsiatwal**
This PR removes the 4 products that were behind a feature flag

### CODE REVIEW

File: exporter/applications/helpers/task_lists.py [hunk 1]
```diff
@@ -93,7 +92,6 @@ def get_application_task_list(request, application, errors=None):
     context["notes"] = get_case_notes(request, application["id"])["case_notes"]
 
     context["security_approvals"] = get_security_approval_details(application)
-    context["FEATURE_FLAG_F680_SECURITY_CLASSIFIED_ENABLED"] = settings.FEATURE_FLAG_F680_SECURITY_CLASSIFIED_ENABLED
```
Comment (currycoder) [inline exporter/applications/helpers/task_lists.py:96]: Could this be removed from `conf/exporter.py` too?

File: unit_tests/exporter/applications/views/goods/component/test_edit_component.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def setup(
     settings,
     no_op_storage,
 ):
-    settings.FEATURE_FLAG_NON_FIREARMS_COMPONENT_ENABLED = True
+    pass
```
Comment (currycoder) [inline unit_tests/exporter/applications/views/goods/component/test_edit_component.py:22]: Could this whole setup function be removed now?

File: unit_tests/exporter/applications/views/goods/material/test_edit_material.py [hunk 3]
```diff
@@ -18,7 +18,7 @@ def setup(
     settings,
     no_op_storage,
 ):
-    settings.FEATURE_FLAG_NON_FIREARMS_MATERIAL_ENABLED = True
+    pass
```
Comment (currycoder) [inline unit_tests/exporter/applications/views/goods/material/test_edit_material.py:21]: Same again

File: unit_tests/exporter/applications/views/goods/platform/test_edit_platform.py [hunk 4]
```diff
@@ -18,7 +18,7 @@ def setup(
     settings,
     no_op_storage,
 ):
-    settings.FEATURE_FLAG_NON_FIREARMS_PLATFORM_ENABLED = True
+    pass
```
Comment (currycoder) [inline unit_tests/exporter/applications/views/goods/platform/test_edit_platform.py:21]: And again

File: unit_tests/exporter/applications/views/goods/software/test_edit_software.py [hunk 5]
```diff
@@ -13,7 +13,7 @@ def setup(
     mock_control_list_entries_get,
     settings,
 ):
-    settings.FEATURE_FLAG_NON_FIREARMS_SOFTWARE_ENABLED = True
+    pass
```
Comment (currycoder) [inline unit_tests/exporter/applications/views/goods/software/test_edit_software.py:16]: :D

File: unit_tests/exporter/applications/views/test_add_good.py [hunk 6]
```diff
@@ -32,8 +32,8 @@
 
 
 @pytest.fixture(autouse=True)
-def setup(no_op_storage, settings):
-    settings.FEATURE_FLAG_NON_FIREARMS_ENABLED = False
+def setup(no_op_storage):
+    pass
```
Comment (currycoder) [inline unit_tests/exporter/applications/views/test_add_good.py:36]: :raised_hands:

File: exporter/applications/helpers/task_lists.py (hunk 1 above)
Comment (depsiatwal) [reply exporter/applications/helpers/task_lists.py:96]: well spotted i've actually reverted these changes back as I decided to do FEATURE_FLAG_F680_SECURITY_CLASSIFIED_ENABLED in another PR.

File: unit_tests/exporter/applications/views/goods/component/test_edit_component.py (hunk 2 above)
Comment (depsiatwal) [reply unit_tests/exporter/applications/views/goods/component/test_edit_component.py:22]: some of the setup items happen via parameters for example 
no_op_storage
is setup for every test.

Comment (currycoder) [reply unit_tests/exporter/applications/views/goods/component/test_edit_component.py:22]: Got it - thanks for explaining.  Would you mind popping a singe line comment in each spot to explain this?  If I were going through this I would probably delete a function whose body is just `pass`

File: exporter/applications/helpers/task_lists.py (hunk 1 above)
Comment (currycoder) [reply exporter/applications/helpers/task_lists.py:96]: But this is still removed?  Doesn't it need to be re-instated in this PR?

File: unit_tests/exporter/applications/views/goods/component/test_edit_component.py (hunk 2 above)
Comment (currycoder) [reply unit_tests/exporter/applications/views/goods/component/test_edit_component.py]: That's a good solution, thanks

File: exporter/applications/helpers/task_lists.py [hunk 7]
```diff
@@ -93,8 +93,8 @@ def get_application_task_list(request, application, errors=None):
...
 
+    context["FEATURE_FLAG_F680_SECURITY_CLASSIFIED_ENABLED"] = settings.FEATURE_FLAG_F680_SECURITY_CLASSIFIED_ENABLED
```
Comment (depsiatwal) [inline exporter/applications/helpers/task_lists.py:97]: oh it has been reinstated it's just moved down to line 97

**Review by currycoder — APPROVED** [review]

**Pull request merged**